### PR TITLE
Update gradle to `7.6` + Rollback android core to `1.6.0`

### PR DESCRIPTION
**Pull request by FirelightFlagboy**

### CODE REVIEW

File: oxidation/client/android/app/build.gradle [hunk 1]
```diff
@@ -1,6 +1,8 @@
 apply plugin: 'com.android.application'
 
 android {
+    namespace "com.scille.parsec"
```
Comment (touilleMan) [inline oxidation/client/android/app/build.gradle:4]: What is this for ?

File: oxidation/client/android/app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.scille.parsec">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
```
Comment (touilleMan) [inline oxidation/client/android/app/src/main/AndroidManifest.xml:2]: What is this for ?

File: oxidation/client/android/app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -15,7 +14,8 @@
             android:name="com.scille.parsec.MainActivity"
             android:label="@string/title_activity_main"
             android:theme="@style/AppTheme.NoActionBarLaunch"
-            android:launchMode="singleTask">
+            android:launchMode="singleTask"
+            android:exported="false">
```
Comment (touilleMan) [inline oxidation/client/android/app/src/main/AndroidManifest.xml:18]: What is this for ?

File: oxidation/client/android/app/build.gradle (hunk 1 above)
Comment (FirelightFlagboy) [reply oxidation/client/android/app/build.gradle:4]: For this error message

```
package="com.scille.parsec" found in source AndroidManifest.xml: /home/andesite/Documents/parsec-cloud/oxidation/client/android/app/src/main/AndroidManifest.xml.
Setting the namespace via a source AndroidManifest.xml's package attribute is deprecated.
Please instead set the namespace (or testNamespace) in the module's build.gradle file, as described here: https://developer.android.com/studio/build/configure-app-module#set-namespace
```

tl,dr: we need to specify the namespace in `build.gradle`

File: oxidation/client/android/app/src/main/AndroidManifest.xml (hunk 2 above)
Comment (FirelightFlagboy) [reply oxidation/client/android/app/src/main/AndroidManifest.xml:2]: see https://github.com/Scille/parsec-cloud/pull/3732#discussion_r1043201769

File: oxidation/client/android/app/src/main/AndroidManifest.xml (hunk 3 above)
Comment (FirelightFlagboy) [reply oxidation/client/android/app/src/main/AndroidManifest.xml:18]: It's to correct a deprecation error

```
/home/foobar/Documents/parsec-cloud/oxidation/client/android/app/src/main/AndroidManifest.xml:13:9-25:20 Error:
        android:exported needs to be explicitly specified for element <activity#com.scille.parsec.MainActivity>. Apps targeting Android 12 and higher are required to specify an explicit value for `android:exported` when the corresponding component has an intent filter defined. See https://developer.android.com/guide/topics/manifest/activity-element#exported for details.
```

tl,dr: the value `android:exported` need to be explicitly set

**Review by touilleMan — APPROVED** [review]

**Pull request merged**